constants/constants.go

```diff
@@ -0,0 +1,10 @@
+package constants
+
+const (
+	// ComponentNameKey is the tag key identifying the service
+	ComponentNameKey = "lightstep.component_name"
```

Comment:
can we re-use https://github.com/lightstep/lightstep-tracer-go/blob/master/options.go#L39?

Reply:
i created the constants package to allow both metrics and the lightstep package to share constants. otherwise it was creating an import cycle when trying to import options from the metrics package

Reply:
Thoughts on sticking these into an `internal` package? (Moving them out of the top package would already constitute a breaking change.)